src/Forms/TurnstileCaptchaField.php

```diff
@@ -2,50 +2,53 @@
 
 namespace Terraformers\TurnstileCaptcha\Forms;
 
+use GuzzleHttp\Exception\RequestException;
+use Locale;
 use Psr\Log\LoggerInterface;
 use SilverStripe\Control\Controller;
+use SilverStripe\Core\Environment;
 use SilverStripe\Core\Injector\Injector;
 use SilverStripe\Forms\FormField;
 use SilverStripe\i18n\i18n;
 use SilverStripe\View\Requirements;
-use Locale;
+use Terraformers\TurnstileCaptcha\Http\HttpClient;
 
 class TurnstileCaptchaField extends FormField
 {
     /**
      * Recaptcha Site Key
-     * @config NocaptchaField.site_key
+     * @config TurnstileCaptchaField.site_key
      */
     private static ?string $site_key = null;
 
     /**
      * Recaptcha Secret Key
-     * @config NocaptchaField.secret_key
+     * @config TurnstileCaptchaField.secret_key
      */
     private static ?string $secret_key = null;
 
 
     /**
      * CURL Proxy Server location
-     * @config NocaptchaField.proxy_server
+     * @config TurnstileCaptchaField.proxy_server
```

Comment:
these properties (Proxy/ssl) can be removed as the client class is where that should happen